### PR TITLE
Add Cypher Core Public API

### PR DESCRIPTION
**Pull request by jim380**

### CODE REVIEW

File: cosmoshub/chain.json [hunk 1]
```diff
@@ -288,6 +288,10 @@
       {
         "address": "cosmos-grpc.polkachu.com:14990",
         "provider": "Polkachu"
+      },
+      {
+        "address": "https://cosmos-api.cyphercore.io",
```
Comment (thomasWos) [inline cosmoshub/chain.json:293]: This is the grpc definition, rest API instead?

Comment (jim380) [reply cosmoshub/chain.json]: @thomasWos Fixed. Thanks for catching that.

**Comment by JeremyParish69** [comment]
Looks like there are conflicts now

**Comment by jim380** [comment]
@JeremyParish69 conflicts resolved. Thanks!

**Pull request merged**